custom_components/powercalc/data/wiz/SHTW/model.json

```diff
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Kasa EP25",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.8.1:docker"
+    },
+    "name": "SHTW",
+    "standby_power": 7.12
```

Comment:
This can't be correct. When it is correct I would throw the light in the garbage can immediately ;-)